zilliqa/src/db.rs

```diff
@@ -231,11 +266,11 @@ impl Db {
                 gas_limit INTEGER NOT NULL,
                 qc BLOB NOT NULL,
                 agg BLOB,
-                is_canonical BOOLEAN NOT NULL);
+                is_canonical BOOLEAN NOT NULL) WITHOUT ROWID;
```

Comment:
What's the downside of `WITHOUT ROWID`?

Reply:
From the docs, as long as the Primary Key is not too big, then there is not much of a downside. The recommendation is to keep it under 1/20th of a page size. In our case, the page size is configured as 32KB (4KB default), which would be 1.6KB (200 bytes default). In either case, our Primary Key is under that size, iirc.